windows.Dockerfile

```diff
@@ -1,11 +1,16 @@
-FROM mcr.microsoft.com/windows/servercore:1809 as core
+FROM --platform=$BUILDPLATFORM golang:1.13.10-alpine3.10 as builder
+WORKDIR /go/src/github.com/Azure/secrets-store-csi-driver-provider-azure
+ADD . .
+ARG TARGETARCH
+ARG TARGETOS
+ARG LDFLAGS
+RUN CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -a -ldflags "${LDFLAGS}" -o _output/secrets-store-csi-driver-provider-azure.exe ./cmd/
 
 FROM mcr.microsoft.com/windows/servercore:ltsc2019
```

Comment:
Is there a reason for `ltsc2019`?

Reply:
Updated to use `mcr.microsoft.com/powershell:lts-nanoserver-1809`